packages/firestore/src/protos/firestore_proto_api.ts

```diff
@@ -382,6 +391,7 @@ export declare namespace firestoreV1ApiClientInterfaces {
     readTime?: Timestamp;
     targetId?: number;
     once?: boolean;
+    expectedCount?: number;
```

Comment:
I don't think that `number` is correct here, since `expectedCount` is a `google.protobuf.Int32Value`. Here is the definition of `Int32Value`:

https://github.com/firebase/firebase-js-sdk/blob/e9fb4037f3908d2a05070de9219aa8a94d287f14/packages/firestore/src/protos/google/protobuf/wrappers.proto#L85-L91